src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1423,7 +1423,9 @@ namespace winrt::TerminalApp::implementation
                 }
             }
         });
-        _AppendCloseMenuItems(contextMenuFlyout);
+        auto closeSubMenu = _AppendCloseMenuItems(contextMenuFlyout);

```

Comment:
I am a little bit weirded out by this returning a menu into which we then insert another thing, but also. whatever. that's cool :)